overrides/scripts/MultiblockSimulationChamber.zs

```diff
@@ -0,0 +1,280 @@
+};
+
+sim_chamber_multiblock.completeRecipe = function(logic as IRecipeLogic) {
+    for slot, stack in logic.inputInventory {
+        if(!isNull(stack) && stack.definition.id.startsWith(model_prefix)) {
+            var tier = asInt(stack.tag.tier);
+            var simulationCount = asInt(stack.tag.simulationCount) + 1;
+            var killCount = asInt(stack.tag.killCount);
+
+            if(tier < DATA_MODEL_MAXIMUM_TIER) {
+                val roof = maxExperience[tier];
+
+                if(simulationCount + 1 >= roof) {
+                    killCount = 0;
+                    simulationCount = 0;
+                    tier += 1;
+                }
+            }
+
+            logic.inputInventory.setStackInSlot(
+                slot,
+                stack.withTag(
+                    stack.tag + ({
+                        tier: tier,
+                        simulationCount: simulationCount,
+                        totalSimulationCount: asInt(stack.tag.totalSimulationCount) + 1,
+                        killCount: killCount
+                    } as IData)
+                )
+            );
+            return;
+        }
+    }
+} as ICompleteRecipeFunction;
+
+val withTier = function(tier as int) as IItemCondition {
+    return function(stack as IItemStack) as bool {
+        return asInt(stack.tag.tier) == tier;
+    } as IItemCondition;
+};
+
+for mob, cost in mobs {
+    for tier, pristine_chance in pristine_chances {
+        dml_recipe_map.recipeBuilder()
+            .duration(301)
+            .EUt(cost / 4 as int)
+            .inputs(<deepmoblearning:polymer_clay>)
+            .notConsumable(itemUtils.getItem(model_prefix + mob).withTag({tier: tier + 1}, false).only(withTier(tier + 1)))
+            .outputs(pristine_types[mob])
+            .chancedOutput(itemUtils.getItem(pristine_prefix + mob), pristine_chance * 100, 0)
+            .buildAndRegister();
+    }
+}
+
+
+//Recipe for Naquadah-Plated Dark Steel Casing
+makeShaped("simulation_casing", <contenttweaker:simulation_casing>,
+    ["SPS",
+     "PCP",
+     "SPS"],
+    { S : <metaitem:screwNaquadahAlloy>,
+      P : <metaitem:plateNaquadahAlloy>,
+      C : <deepmoblearning:machine_casing>
+      });
+
+//Recipe for Controller
+makeShaped("simulation_controller", <gregtech:machine:2003>,
+    ["MHM",
+     "RCR",
+     "MEM"],
+    { M : <metaitem:circuit.wetware_mainframe>,
+      R : <metaitem:robot.arm.uv>,
+      C : <contenttweaker:simulation_casing>,
+      E : <draconicevolution:draconic_energy_core>,
+      H : <contenttweaker:heartofauniverse>
+      });
+
+makeShaped("simulation_casing_two", <contenttweaker:simulation_casing_2>,
+    ["III",
+     "IWI",
+     "III"],
+    { I : <metaitem:plateNaquadah>,
+      W : <ore:gtceWrenches>
+      });
+
+assembler.recipeBuilder()
+    .inputs(<metaitem:plateNaquadah> * 4)
+    .outputs(<contenttweaker:simulation_casing_2>)
+    .duration(200).EUt(30).buildAndRegister();
```

Comment:
Nitpick: possible conflict with naquadah rotors.
Not that anyone would ever use them though.

Reply:
I'm guessing you mean Turbine Blades? Those uniquely use Circuit 10 in the recipe though so there should be extremely little risk of recipe overlap.

Reply:
Issues may come from the fact that assembling machines may erroneously pick the naquadah casing recipe when making rotors, akin to #350. But then again, I don't think anyone would ever make naquadah blades, so it's mostly just a nitpick.

Reply:
I think it's not really any more of a problem than the 6 plate + 1 frame machine casing recipes from GTCE that don't use a disambiguating circuit. Those could cause problems in theory if you tried to use a Circuit 8 in it or something, but blocking mode largely prevents such issues (and it's completely avoidable by not trying to run possibly conflicting recipes from the same machine).